axelor-base/src/main/resources/views/Partner.xml

```diff
@@ -986,6 +981,11 @@
    	<action-attrs name="action-partner-attrs-user-group" if-module="axelor-crm">
 		<attribute name="domain" for="user" expr="eval: &quot;self.group IN (${__config__.app.getApp('crm').groupsAssignable?.collect{it.id}.join(',')})&quot;" if="__config__.app.getApp('crm').assignableUsers == 2"/>
 	</action-attrs>
+
+	<action-attrs name="action-partner-attrs-set-issubcontactor">
+		<attribute name="value" for="$isSubcontractor" expr="eval:false" if="!(__config__.app.isApp('production') &amp;&amp; __config__.app.getApp('production').getManageOutsourcing())"/>
```

Comment:
do it directly, expr="eval: __config__.app.isApp('production')?.getManageOutsourcing()"